osf/metrics/reporters/institution_summary_monthly.py

```diff
@@ -0,0 +1,93 @@
+from django.contrib.contenttypes.models import ContentType
+from django.db.models import Q, F, Sum
+
+from osf.models import Institution, Preprint, AbstractNode, FileVersion
+from osf.models.spam import SpamStatus
+from addons.osfstorage.models import OsfStorageFile
+from osf.metrics.reports import InstitutionMonthlySummaryReport
+from osf.metrics.utils import YearMonth
+from ._base import MonthlyReporter
+from datetime import datetime
+
+
+class InstitutionalSummaryMonthlyReporter(MonthlyReporter):
+    """Generate an InstitutionMonthlySummaryReport for each institution."""
+
+    def report(self, yearmonth: YearMonth):
+        for institution in Institution.objects.all():
+            yield self.generate_report(institution, yearmonth)
+
+    def generate_report(self, institution, yearmonth):
+        node_queryset = institution.nodes.filter(deleted__isnull=True)
```

Comment:
should exclude spam, and (since reporters are sometimes run retroactively) filter `created__lt=yearmonth.next_month()`